tools/yamlgen/main.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2021 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/ceph/ceph-csi/api/deploy/ocp"
+)
+
+const header = `---
+#
+# /!\ DO NOT MODIFY THIS FILE
+#
+# This file has been automatically generated by Ceph-CSI yamlgen.
+# The source for the contents can be found in the api/deploy directory, make
+# your modifications there.
+#`
+
+func main() {
+	fmt.Println(header)
+	fmt.Print(ocp.NewSecurityContextConstraintsYAML(ocp.SecurityContextConstraintsDefaults))
```

Comment:
can we directly write files to a specific location here itself instead of printing and redirecting the output in a makefile?

Reply:
yeah, I was planning that for a follow-up, but can include a start of that now too.

Reply:
if we do that we are good to go on this one from my point.